### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by rhrmo**
Created unit tests for EVF.

https://issues.redhat.com/browse/CCXDEV-10299

### CODE REVIEW

File: pkg/errata/handler.go [hunk 1]
```diff
@@ -45,10 +46,9 @@ func New(url, krb5ConfFile, username, realm, password string) (*Handler, error)
 	}, nil
 }
 
-func (h *Handler) getErrata(id string) ([]byte, error) {
+func (h *Handler) getErrata(ctx context.Context, id string) ([]byte, error) {
 	url := fmt.Sprintf("%s/%s.json", h.url, id)
-	//TODO use request with context
-	r, _ := http.NewRequest("GET", url, nil)
+	r, _ := http.NewRequestWithContext(ctx, "GET", url, nil)
```
Comment (tremes) [inline pkg/errata/handler.go:51]: nit: We could use `http.MethodGet`

File: pkg/errata/handler_test.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		}
+	}
+}`
+	tests := []struct {
+		name       string
+		writerJson string
+		exp        string
+	}{
+		{
+			name:       "rhsa",
```
Comment (tremes) [inline pkg/errata/handler_test.go:36]: This is not very meaningful name for test :) IMO

File: pkg/errata/handler_test.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+			writerJson: rhsa,
+			exp:        "Moderate: OpenShift Container Platform 4.12.5 security update",
+		},
+		{
+			name:       "rhba",
+			writerJson: rhba,
+			exp:        "Moderate: OpenShift Container Platform 4.12.5 security update",
+		},
```
Comment (tremes) [inline pkg/errata/handler_test.go:44]: Perhaps we could also test case when none of these are available?

File: pkg/errata/handler_test.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	}
+
+	for _, test := range tests {
```
Comment (tremes) [inline pkg/errata/handler_test.go:47]: Better to wrap each test to `t.Run(test.name, func(....`

File: pkg/errata/handler_test.go [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/jcmturner/gokrb5.v7/client"
+	"gopkg.in/jcmturner/gokrb5.v7/config"
+	"gopkg.in/jcmturner/gokrb5.v7/spnego"
+)
+
+func Test_Synopsis(t *testing.T) {
```
Comment (tremes) [inline pkg/errata/handler_test.go:15]: nit: I think it's more common to name it `TestSynopsis`

File: pkg/jira/client_test.go [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getIssue(t *testing.T) {
```
Comment (tremes) [inline pkg/jira/client_test.go:13]: This test for `GetAllIssues` method right?

Comment (rhrmo) [reply pkg/jira/client_test.go:13]: yes, I forgot to rename it, thanks!

**Comment by tremes** [comment]
Thanks
